### PR TITLE
Release v1.6.5 scope review

### PR DESCRIPTION
**Pull request by creisle**
New Features
- overlay read depth plot density

Improvements
- exon drawing, fixed overlapping exon issues
- additional columns to summary output

### CODE REVIEW

File: docs/source/tools_supported.rst [hunk 1]
```diff
@@ -128,3 +130,23 @@ You can also re-use the same conversion script if you have multiple inputs to co
...
+
+Assuming that the tool outputting the VCF file follows standard conventions, then it is possible to use a general VCF 
+conversion that is not tool-specific. Given the wide variety in content for VCF files, MAVIS makes a number of
+assumptions and the VCF conversion may not work for all VCFs. In general MAVIS follows the `VCF 4.2 specification <https://samtools.github.io/hts-specs/VCFv4.2.pdf>`_.
+If the input tool you are using differs, it would be better to use a :ref:`custom conversion script <custom-conversion>`.
+
+**Assumptions on non-standard INFO fields**
+
+- ``PRECISE`` if given, Confidence intervals are ignored if given in favour of exact breakpoint calls using pos and END as the breakpoint positions
+- ``CT`` values if given are representative of the breakpoint orientations.
```
Comment (dustinbleile) [inline docs/source/tools_supported.rst]: Note: I'm quite sure 'CT' stands for 'connection type' in DELLY's documentation.  Can be useful when looking things up.

FYI - from https://groups.google.com/forum/#!topic/delly-users/6Mq2juBraRY
Hi Daniel,

These connection types (CT) are probably most relevant for complex re-arrangements in cancer. Another way to think about these is in terms of the paired-end orientation (Medvedev et al. 2010: PMID: 2080529, Figure 3). Different names exist for these connection types in the literature, e.g. Stephens et al. (PMID: 21215367) used head-to-head inversions, tail-to-tail inversions, and so on.

Intra-chromosomal rearrangements:
(1) deletion-type = tail-to-head = 3' to 5' = [+-]
(2) tandem-duplication type = head-to-tail = 5' to 3' = [-+]
(3) left-spanning inversion type = tail-to-tail = 3' to 3' = [++]
(4) right-spanning inversion type = head-to-head = 5' to 5' = [--]

For inter-chromosomal rearrangements the pairs can also map [+-], [-+], [++] and [--] just on 2 different chromosomes and hence, you have all 4 possible connections types for translocations in Delly. The easiest example is a single chromosome A where a virus B integrated. If B was integrated in forward orientation you will see a 3' to 5' translocation and a 5' to 3' translocation at the integration site. If B was integrated in inverted orientation you will see a 3' to 3' translocation and a 5' to 5' translocation at the integration site.

Best, Tobias

File: mavis/config.py [hunk 2]
```diff
@@ -306,18 +306,18 @@ def __init__(self, **kwargs):
                 continue
             val = [v for v in re.split(r'[;\s]+', val) if v]
```
Comment (dustinbleile) [inline mavis/config.py]: if the val list is every empty this would crash.  Is that impossible?

File: docs/source/tools_supported.rst (hunk 1 above)
Comment (creisle) [reply docs/source/tools_supported.rst]: I actually link to that very same google group in the code itself on my CT comments. But here I am talking about what they mean in terms of mavis conversions. We convert connection type (CT) to breakpoint orientations

File: mavis/config.py (hunk 2 above)
Comment (creisle) [reply mavis/config.py]: no, the parser requires 3-5 values for that option

Comment (creisle) [reply mavis/config.py]: true. It should crash then anyway but I could make it a more helpful error message by checking and actually throwing an error

**Pull request merged**